src/DynamoCore/Models/DynamoModelCommands.cs

```diff
@@ -52,19 +52,19 @@ protected virtual void OpenFileImpl(OpenFileCommand command)
             //ClipBoard.Clear();
         }
 
-        void RunCancelImpl(RunCancelCommand command)
+        internal void RunCancelImpl(RunCancelCommand command)
```

Comment:
@alfarok - did you checkout the link @QilongTang sent - I interpret this to mean these should have been private by default?